index.js

```diff
@@ -1,67 +1,200 @@
     },
     {
         title: 'Heartbeats',
-        director: ['Xavier Dolan']
+        director: 'Xavier Dolan',
+        year: 2011,
+        genre: ['Romance','Comedy', 'Drama'],
+        description: `Francis (Xavier Dolan) is a young gay man, Marie (Monia Chokri) is a young straight woman and the two of them are best friends -- until the day the gorgeous Nicolas (Niels Schneider) walks into a Montreal coffee shop. The two friends, instantly and equally infatuated, compete for Nicolas' indeterminate affections, a conflict that climaxes when the trio visit the vacation home of Nicolas' mother. The frothy comedy unfolds through narrative, fantasy sequences and confessional monologues.`,
+        imageUrl: 'https://flxt.tmsimg.com/assets/p8153637_p_v10_ac.jpg'
     },
     {
         title: 'Paris Is Burning',
-        director: ['Jennie Livingston']
+        director: 'Jennie Livingston',
+        year: 1990,
+        genre: ['Documentary'],
+        description: 'This documentary focuses on drag queens living in New York City and their "house" culture, which provides a sense of community and support for the flamboyant and often socially shunned performers. Groups from each house compete in elaborate balls that take cues from the world of fashion. Also touching on issues of racism and poverty, the film features interviews with a number of renowned drag queens, including Willi Ninja, Pepper LaBeija and Dorian Corey.',
+        imageUrl: 'https://flxt.tmsimg.com/assets/p54287_p_v10_ad.jpg'
     },
     {
         title: 'Disclosure',
-        director: ['Sam feder']
+        director: 'Sam Feder',
+        year: 2020,
+        genre: ['Documentary'],
+        description: `An in-depth look at Hollywood's depiction of transgender people and the impact of those stories on transgender lives and American culture.`,
+        imageUrl: 'https://flxt.tmsimg.com/assets/p18415918_p_v13_aa.jpg'
     },
     {
         title: 'Passing',
-        director: ['Rebecca Hall']
+        director: 'Rebecca Hall',
+        year: 2021,
+        genre: ['Drama'],
+        description: `In 1920s New York City, a Black woman finds her world upended when her life becomes intertwined with a former childhood friend who's passing as white.`,
+        imageUrl: 'https://flxt.tmsimg.com/assets/p20588150_p_v10_aa.jpg'
     }
 ]
 
 app.use(morgan('common'));
 
-
-
 app.use(express.static('public',options));
 
 // GET requests
 app.get('/',(req,res)=>{
     res.send("<h1>This is just a default textual response. Nothing to worry about.</h1>")
 })
 
+// get all movies
 app.get('/movies',(req,res)=>{
-    res.json(tenMovies)
+    res.json(movies)
+})
+
+// get specific movie
+app.get('/movies/:title',(req,res)=>{
+    res.json(movies.find((movie)=>{
+        return movie.title ===req.params.title
+    }))
+})
+
+// get genre description by name
+app.get('/genres/:name',(req,res)=>{
+    res.send('This is a description about the film genre ' + req.params.name)
+})
```

Comment:
Can we get the description of a genre correctly please rather than hard coding?

Reply:
done